### ISSUE TITLE
Add support for Firefox

### ISSUE THREAD

**Issue by mirdaki** [microsoft/vscode-dev-chrome-launcher#16]
It'd be great to support Firefox as well. The extension scheme is similar enough it shouldn't add too much complexity. 

I will work on it this week

**Comment by joyceerhl** [comment]
Good idea, I've assigned this issue to you for now :) We'd need to figure out how to publish extensions for Firefox as well.

**Comment by infinitepower18** [comment]
Why is this extension not on Microsoft's own Edge extension store?
